docs/blog/posts/using-vega-protocol-with-hummingbot/index.md

```diff
@@ -0,0 +1,172 @@
+
+[![image](image7.png)](image7.png)
+
+You'll then need to approve the transaction in Metamask - click "Confirm" to confirm the transaction. 
+
+[![image](image8.png)](image8.png)
+
+Once this is done, you'll just need to wait for the transaction to be confirmed. You should get a confirmation message once this is done. 
+
+[![image](image9.png)](image9.png)
+
+
+## Install Hummingbot
+
+There are two main methods to install Hummingbot:
+
+### Docker
+
+For most new users, we recommend installing Hummingbot using Docker.
+
+- [Docker Quickstart Guide](https://hummingbot.org/academy-content/docker-installation-guide/)
+
+### Source
+
+We recommend installing Hummingbot from source if you meet any of the following criteria:
+
+- You want to customize or extend the Hummingbot codebase.
+- You want to build new components like connectors or strategies.
+- You want to learn how Hummingbot works at a deeper, technical level.
+
+Guides for various environments:
+  - [Linux](../../../installation/linux.md)
+  - [Windows](../../../installation/windows.md)
+  - [MacOS](../../../installation/mac.md)
+
+
+## Connect your Vega Wallet to Hummingbot
+
+In order to connect your Vega Wallet to Hummingbot you'll need your `SNAP KEY` (also known as Vega Party ID) and your `wallet seed phrase`. 
+
+Launch Hummingbot and then enter the command below in the Hummingbot terminal
+
+`connect vega_perpetual`
+
+or if you are connecting to the testnet
+
+`connect vega_perpetual_testnet`
+
+You'll be prompted to enter in your Vega public key (party ID) - this can be found in the top right of the screen where it says "SNAP KEY". 
+
+Next, enter in the wallet seed phrase and if both credentials are valid you should get the message 
+"You are now connected to Vega Perpetual"
+
+[![image](image4.png)](image4.png)
+
+Once connected to Hummingbot, you should then be able to confirm it's connected by running the `balance` command
+
+## Starting Your First Trade
+
+`v2_dman_v3_multiple_exchanges.py`
+
+Due to the unconventional method that Vega displays the tokens (instead of BASE-QUOTE), the existing scripts will need to be modified  
+
+**How to modify the script**
+
+Open the `v2_dman_v3_multiple_exchanges.py` using your preferred text editor or IDE like VSCode and make changes to the script. For example you can change the trading pair below or change the source exchange for the Candles Feed. 
+
+[![image](image10.png)](image10.png)
+
+
+!!! Note
+    When using a public node, you may encounter issues with the node getting disconnected from time to time which may prompt some errors in the log pane. Hummingbot normally recovers from these issues but for best performance we recommend running a private node instead. 
+
+## Additional Resources
+
+- Vega Mainnet: [Explore Markets](https://vegaprotocol.eth.limo/#/markets)
```

Comment:
Leave a placeholder for Vega to add a blurb or resources that they would users to know. 

Before publication, we should post this user guide PR in the Vega channel for them to comment as well